src/uwtools/tests/drivers/test_esg_grid.py

```diff
@@ -0,0 +1,140 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+EsgGrid driver tests.
+"""
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+import f90nml  # type: ignore
+import yaml
+from pytest import fixture
+
+from uwtools.drivers import esg_grid
+from uwtools.scheduler import Slurm
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "esg_grid": {
+            "execution": {
+                "batchargs": {
+                    "export": "NONE",
+                    "nodes": 1,
+                    "stdout": "/path/to/file",
+                    "walltime": "00:02:00",
+                },
+                "envcmds": [
+                    "module load some-module",
+                    "module load esg-grid-module",
+                ],
+                "executable": "/path/to/esg_grid",
+                "mpiargs": ["--export=ALL", "--ntasks $SLURM_CPUS_ON_NODE"],
+                "mpicmd": "srun",
+            },
+            "namelist": {
+                "update_values": {
+                    "regional_grid_nml": {
+                        "delx": 0.11,
+                        "dely": 0.11,
+                        "lx": -214,
+                        "ly": -128,
+                        "pazi": 0.0,
+                        "plat": 38.5,
+                        "plon": -97.5,
```

Comment:
Here, too, is this supposed to be `plon` or `plot`? If it's `plon` it's out-of-order; if it's `plot` it's misspelled :)

Reply:
The YAML options for the namelist should be: delx, dely, lx, ly, pazi, plat, and plon. I'll update the YAML in `docs/shared` for this entry.

Reply:
My mistake, `plon` is not out-of-order!